doc/manual/src/language/advanced-attributes.md

```diff
@@ -345,3 +345,15 @@ Derivations can declare some infrequently used optional attributes.
     This is useful, for example, when generating self-contained filesystem images with
     their own embedded Nix store: hashes found inside such an image refer
     to the embedded store and not to the host's Nix store.
+
+- [`requiredSystemFeatures`]{#adv-attr-requiredSystemFeatures}\
+
+  If a derivation has the `requiredSystemFeatures` attribute, then Nix will only build it on a machine that has the corresponding features set in its [`system-features` configuration](@docroot@/command-ref/conf-file.md#conf-system-features).
```

Comment:
These have to be repeated in the remote builder configuration as well; for remote builds it won't look at the remote's configuration, [which is unfortunate but also simpler and low latency.]

Reply:
This is stuff for the remote builds reference page. That needs to be reworked anyway.

Reply:
Doesn't mean that we can present false information here.

I suppose you could say
```suggestion
  If a derivation has the `requiredSystemFeatures` attribute, then Nix will only build it locally if the [`system-features` configuration](@docroot@/command-ref/conf-file.md#conf-system-features) contains the corresponding features.
```

although that's still partial information to a degree that it's confusing. Best to just mention remote builds here.